src/components/DndTable/Table.js

```diff
@@ -318,28 +290,32 @@ class EnhancedTable extends React.Component {
   columnRender = () => {
    if (this.state.columnDataCopy.length > 0) {
       return this.state.columnDataCopy;
-    } else return this.state.columnData;
+    } else if (this.props.name === "Transaction"){
+      return this.state.columnData
+    } 
+    else {
+      return this.state.columnData;
+    }
   };
 
   render() {
-    const { classes, data } = this.props;
+    const { classes, data, name } = this.props;
     const { order, orderBy, selected, rowsPerPage, page, renderer } =
       this.state;
     const emptyRows =
       rowsPerPage - Math.min(rowsPerPage, data.length - page * rowsPerPage);
     return (
       <Paper className="searchBox">
-        <DatePicker
-          setEndDate={(date) => this.setEndDate(date)}
-          setStartDate={(date) => this.setStartDate(date)}
-        />
-
         <EnhancedTableToolbar
-          title="Transaction"
+          title={name}
           numSelected={selected.length}
           items={this.props.data}
           searchedData={this.searchedFunc}
         />
+        <DatePicker
+          setEndDate={(date) => this.setEndDate(date)}
+          setStartDate={(date) => this.setStartDate(date)}
```

Comment:
you can rewrite it as
setEndDate={this.setEndDate}

Reply:
done